tests/python/frontend/tensorflow/test_forward.py

```diff
@@ -2866,10 +2866,49 @@ def test_forward_add_n():
     _test_forward_add_n(in5)
 
 
+def _test_dilation2d(opname, tensor_in_sizes, filter_in_sizes,
+                     strides, rates, padding,
+                     deconv_output_shape=[]):
+    """ One iteration of dilation2d with given shapes and attributes """
+
+    total_size_1 = np.prod(tensor_in_sizes)
+    total_size_2 = np.prod(filter_in_sizes)
+    # Initializes the input tensor with array containing incrementing
+    # numbers from 1.
+    data_array = [f * 1.0 for f in range(1, total_size_1 + 1)]
+    filter_array = [f * 1.0 for f in range(1, total_size_2 + 1)]
+
+    with tf.Graph().as_default():
+        in_data = array_ops.placeholder(shape=tensor_in_sizes, dtype='float32')
+        in_filter = constant_op.constant(
+            filter_array, shape=filter_in_sizes, dtype='float32')
+
+        if opname == 'dil2d':
+            nn_ops.dilation2d(in_data,
+                              in_filter,
+                              strides=strides,
+                              rates=rates,
+                              padding=padding)
+
+            compare_tf_with_tvm(np.reshape(data_array, tensor_in_sizes).astype('float32'),
+                                'Placeholder:0', 'Dilation2D:0')
+
+def test_forward_dilation():
+    if is_gpu_available():
```

Comment:
is this op only for GPU?

Reply:
this is no only for GPU, so changed the execution plan without GPU. CI is through now